raco/language/myrialang.py

```diff
@@ -806,6 +838,9 @@ def __repr__(self):
     def num_tuples(self):
         return self._num_tuples
 
+    def partitioning(self):
```

Comment:
I don't quite understand what "getting the partitioning info from the SQL query fragment's plan" means. It looks like the "plan" is just a SQLAlchemy expression, which wouldn't know anything about the partitioning info in the catalog. I'm not suggesting trying to extract partitioning info from `MyriaQueryScan`, which I agree would be intractable, but rather pushing partitioning info into its constructor (https://github.com/uwescience/raco/blob/master/raco/language/myrialang.py#L1473):

```
return MyriaQueryScan(sql=sql_string.process(sql_plan),
                                  scheme=expr.scheme(),
                                  partitioning=expr.partitioning(),
                                  num_tuples=expr.num_tuples())
```